Editorial review: Document element-scoped view transitions (#44217)
mdn/content: files/en-us/web/api/element/activeviewtransition/index.md

---
title: "Element: activeViewTransition property"
short-title: activeViewTransition
slug: Web/API/Element/activeViewTransition
page-type: web-api-instance-property
browser-compat: api.Element.activeViewTransition
---

> {{APIRef("DOM")}}

Reviewer: should this be "{{APIRef("View Transition API")}}" like in `Element/startViewTransition`

Author: That would probably be better, yes. Changed.

> The **`activeViewTransition`** read-only property of the {{domxref("Element")}} interface returns a {{domxref("ViewTransition")}} instance representing the [view transition](/en-US/docs/Web/API/View_Transition_API) currently active on an element. It provides a consistent way to access an active element-scoped view transition without having to worry about saving a reference for easy access later on.

Reviewer: add link on "element-scoped view transition" to `/en-US/docs/Web/API/View_Transition_API/Using_element-scoped`?

```suggestion
The **`activeViewTransition`** read-only property of the {{domxref("Element")}} interface returns a {{domxref("ViewTransition")}} instance representing the [view transition](/en-US/docs/Web/API/View_Transition_API) currently active on an element. It provides a consistent way to access an active element-scoped view transition without needing to save a reference to it for later.
```

Author: updated, here, and in the Document version.

An element-scoped {{domxref("ViewTransition")}} can also be accessed via the return value of {{domxref("Element.startViewTransition()")}}.

## Value

A {{domxref("ViewTransition")}} or `null` if the element has no active view transition.

## Examples

### Basic usage

> 

Reviewer: short description to explain what it's demo'ing?

Author: done

```js
const myElement = document.querySelector(".my-element");

// ...

function handleVT() {
  const vt = myElement.startViewTransition(() => {
    updateDOMSomehow();
  });
}

> // Returns reference to vt, above, provided transition is still ongoing

Reviewer: ```suggestion
// Returns a reference to `vt` if the transition is still ongoing
```

Author: updated
